### ISSUE TITLE
Best Way to Enable USM via Operator and v1alpha1 Spec?

### ISSUE THREAD

**Issue by daviddyball** [DataDog/datadog-operator#657]
**Describe what happened:**

I'd like to enable USM as per documentation [here](https://docs.datadoghq.com/universal_service_monitoring/?tab=kuberneteswithouthelm#enabling-universal-service-monitoring), but it gets a little fuzzy with the `v1alpha1` spec. I can see that `v2alpha1` spec has `features.usm` flag but, as far as I can tell, the `v2alpha1` CRDs aren't yet getting deployed to the cluster.

**Describe what you expected:**

I'd like to find a way to enable USM using the `v1alpha1` spec, or discover how to start using the `v2alpha1` spec.

**Steps to reproduce the issue:**

N/A

**Additional environment details (Operating System, Cloud provider, etc):**

N/A

**Comment by WoutDev** [comment]
I have the same question. Is it possible as of now?

**Comment by CharlyF** [comment]
Hi all - We just released the Datadog Operator 1.0 along with the major `datadog-operator` helm chart update `1.0.0`.
We have the DatadogAgent v2alpha1 stored by default and have [documented here](https://docs.datadoghq.com/universal_service_monitoring/#enabling-universal-service-monitoring) how to get USM enabled:
With the new template, it is as easy as:
```
apiVersion: datadoghq.com/v2alpha1
kind: DatadogAgent
metadata:
  name: datadog
spec:
  global:
    credentials:
     apiSecret:
        secretName: datadog-secret
        keyName: api-key
     appSecret:
      secretName: datadog-secret
      keyName: app-key
  features:
    usm:
      enabled: true
```

It was available in v1alpha1 however, and if you are using the DatadogAgent v1alpha1, please follow the [migration documentation here](https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator#migrating-to-the-version-10-of-the-datadog-operator) to get the new Generally Available version.

**Comment by CharlyF** [comment]
I am going to close this out since my previous answer should be responding to the ask - Do let us know if you have any follow up questions.
